doc/radosgw/cloud-transition.rst

```diff
@@ -190,27 +202,27 @@ For example:
             }
         }
     ]
+```
 
+.. note:: Once a storage class is created
```

Comment:
I think probably this would read more elegantly if it were to say

"Once a storage class of X or Y is created" instead of "Once a storage class is created of X or Y".